spencer_caldwell/package.json

```diff
@@ -0,0 +1,21 @@
+{
+  "name": "spencer_caldwell",
+  "version": "1.0.0",
+  "description": "HTTP Server With Persistence ============================ To complete this assignment:   * fork this repository (the sub module for this specific assignment)   * clone down your fork   * place all of your work in a folder that is your full name, use `_`s instead of spaces   * push back up to your fork   * create a pull request back to the original repo   * submit a link to the PR in canvas",
```

Comment:
The description should be a description of YOUR WORK, not the assignment description.  If you want to keep the assignment description for reference, make a markdown file for it.